### ISSUE TITLE
Failing scenario for `1.13` after updating to `3.0`

### ISSUE THREAD

**Issue by karellm** [ember-cli/ember-try#187]
I'm new to addon development and currently trying to make a PR for an addon. Part of that PR I updated to `ember@3.0`. This caused the `1.13` scenario to fail with:

```
TypeError: Cannot read property 'buildInstance' of undefined
    at application.boot.then.app (http://localhost:7357/assets/test-support.js:7051:49)
    at tryCatch (http://localhost:7357/assets/vendor.js:66698:14)
    at Object.invokeCallback (http://localhost:7357/assets/vendor.js:66713:15)
    at http://localhost:7357/assets/vendor.js:68525:21
    at http://localhost:7357/assets/vendor.js:43055:7
    at invoke (http://localhost:7357/assets/vendor.js:10575:16)
    at Queue.flush (http://localhost:7357/assets/vendor.js:10639:11)
    at DeferredActionQueues.flush (http://localhost:7357/assets/vendor.js:10440:17)
    at Backburner.end (http://localhost:7357/assets/vendor.js:9729:25)
    at http://localhost:7357/assets/vendor.js:10320:18
```

I tried both with the `package.json` definition:

```
    "versionCompatibility": {
      "ember": ">1.13.0 <=3.0.0"
    }
```

and with this following config:

```
{
  useYarn: true,
  scenarios: [
    {
      name: 'ember-1.13',
      bower: {
        dependencies: {
          'ember': '1.13.13'
        }
      },
      npm: {
        devDependencies: {
          'ember-source': null
        }
      }
    },
    {
      name: 'ember-lts-2.12',
      npm: {
        devDependencies: {
          'ember-source': '~2.12.0'
        }
      }
    },
    {
      name: 'ember-lts-2.16',
      npm: {
        devDependencies: {
          'ember-source': '~2.16.0'
        }
      }
    },
    {
      name: 'ember-lts-2.18',
      npm: {
        devDependencies: {
          'ember-source': '~2.18.0'
        }
      }
    },
    {
      name: 'ember-release',
      npm: {
        devDependencies: {
          'ember-source': urls[0]
        }
      }
    },
    {
      name: 'ember-beta',
      npm: {
        devDependencies: {
          'ember-source': urls[1]
        }
      }
    },
    {
      name: 'ember-canary',
      npm: {
        devDependencies: {
          'ember-source': urls[2]
        }
      }
    },
    {
      name: 'ember-default',
      npm: {
        devDependencies: {}
      }
    }
  ]
}
```

Does it ring a bell? Thanks for any help you might be able to provide.

**Comment by rwjblue** [comment]
This is most likely a bug for emberjs/ember-test-helpers, I believe it is testing against 1.13 though so a reproduction would be helpful...

**Comment by karellm** [comment]
@rwjblue Can you checkout this PR here: https://github.com/OCTRI/ember-i18next/pull/52

If you `yarn` and `ember try:each` or `ember try:one ember-1.13.13` you should see the error. Let me know if you need anything else.

**Comment by rwjblue** [comment]
lots of convo in  slack around the failure mentioned here

tldr; this is not an ember-try bug, to fix it would require changes in ember-test-helpers to work around the fact that ember@1.13 did not resolve its `boot` promise with the `app` instance itself (and therefore in `buildOwner` we are calling `undefined.buildInstance()`).

**Comment by karellm** [comment]
Thanks for the help @rwjblue. I'll close this issue since supporting `1.13` is not currently a priority and this is not even an `ember-try` error per say.
